### ISSUE TITLE
Improve rendering of rich reply to images

### ISSUE THREAD

**Issue by HarHarLinks** [SchildiChat/SchildiChat-android#172]
**Is your feature request related to a problem? Please describe.**
![image](https://user-images.githubusercontent.com/2803622/212294449-b9381793-a3e4-4f07-9934-ac2c57002d4d.png)

1. wide images are "cropped"
  - actually I think it always scales to height and flows over to the right
  - note the missing corner rounding on the right
2. why can the rich reply take more horizontal space than the regular image message?

**Describe the solution you'd like**
- maybe it should never upscale beyond 100%?
- should scale to fit the bigger dimension, like it does for narrow images. -> i.e. scale wide images to fit by width rather than height.

**Describe alternatives you've considered**
/

**Additional context**
List of rich reply commits: https://github.com/vector-im/element-android/issues/3731#issuecomment-1330363453

(btw, what happens with videos?)

**Comment by SpiritCroc** [comment]
Proper image rendering on Android is annoying, when wanting to follow the images measures, while handling the case that the original event did not send correct measures. So I went the easy way with fixed height for now, correct. Maybe when I'm motivated I can at some point port the more sophisticated measuring logic I wrote for regular image rendering to replies. This smarter logic basically updates measures after loading the image, so it can be sure about proper measures. As an easier workaround, one could only handle the case where there are measures included in the event source, and where one assumes that these are correct (hint: in practice, they aren't always correct, sometimes width and height are exchanged. also many bridges do not send measures at all).

I think videos are handled just the same as images, but just using the thumbnail picture.
